drivers/SmartThings/zwave-switch/fingerprints.yml

```diff
@@ -898,6 +898,61 @@ zwaveManufacturer:
     manufacturerId: 0x010F
     productType: 0x0102
     deviceProfileName: fibaro-dimmer-2
+#Shelly/Qubino
+  - id: 1120/2/129
+    deviceLabel: Wave 2PM
+    manufacturerId: 0x0460
+    productId: 0x0081
+    productType: 0x0002
+    deviceProfileName: dual-metering-switch
+  - id: 1120/2/131
+    deviceLabel: Wave 1
+    manufacturerId: 0x0460
+    productId: 0x0083
+    productType: 0x0002
+    deviceProfileName: switch-binary-indicator
```

Comment:
the indicator preference here won't work unless you point out which configuration parameter is used here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/zwave-switch/src/preferences.lua#L310